### ISSUE TITLE
^H being sent in at the start of some messages

### ISSUE THREAD

**Issue by jshirley** [MacIrssi/MacIrssi#28]
(This bug is seemingly something with OS X input controls, since it also happens with Colloquy and LimeChat)

Periodically, at the start of every message a ^H sneaks through and is posted. It's really only evident when typing something like "/me goes to the store" or "/win 13" and the ^H blocks that action from happening.

I see the ^H from other mac users that use those other clients, but for some reason much less frequently than it happens with me.  In my case, it's like every 10 instances. If I run a terminal client, like irssi itself, then it never happens (Terminal.app, Safari, Mail, etc never have this behavior).

Thanks,
-Jay

**Comment by ghost** [comment]
I don't think I've ever seen this happen to me. The only way I'd be able to look at this is if you had full reproduction steps. Even then, it sounds like it might be nothing to do with MacIrssi.

**Comment by jshirley** [comment]
I don't think it has anything to do with MacIrssi, but it certainly is nice to not have phantom ^Hs flittering about.

In my work IRC channel, those who use Macs + Limechat or Colloquy have it happen, but it's only visible to console client users.  When I was using MacIrssi for a bit, it was also happening (but yeah, no reproduction steps – I've been trying for 2+ years to figure it out.  It just happens more frequently on my system than it does on others)

I actually went and forked Limechat and wrote a patch that removes control characters. It's a little too aggressive, but it works for my purposes.

Here's the Limechat patch:
https://gist.github.com/709421
